src/components/new-request/name-check/name-check-tab-content.vue

```diff
@@ -0,0 +1,100 @@
+<template>
+  <v-row class="check-tab-content" justify="center" no-gutters>
+    <v-col>
+      <span class="check-tab-title">{{ title }}</span>
+      <div v-if="loading" class="pt-2" no-gutters>
```

Comment:
Does "no-gutters" do anything on a div?

Reply:
nope - will update (this was a v-row at one point haha)